### ISSUE TITLE
Test if prov vis works with current version of d3 and lodash

### ISSUE THREAD

**Issue by flekschas** [parklab/refinery-platform#734]
In order to avoid using multiple or outdated versions of D3 and Lodash and have a consistent way of loading libraries (i.e. Bower) it'd be great to assess whether the prov vis works with the latest version of D3 (currently 3.5.6) and Lodash (currently 3.10.1)

**Comment by sluger** [comment]
This has been addressed in 5f94e1d83aa2e7575a665015bc540880e4569be7.

**Comment by flekschas** [comment]
The old version (v3.4.11) of d3 is still being loaded on the dataset page, so the prov vis doesn't run with the latest version of d3 (v3.5.6)

https://github.com/parklab/refinery-platform/blob/develop/refinery/templates/core/data_set.html#L632

To reproduce:
1. Go to https://refinery-dev.stemcellcommons.org/data_sets/20651210-9cf9-4c68-9ece-4301de84ad49/#/files/browse
2. Open the developer tool bar
3. Go to the console and type `d3`

This will output:
```
Object {version: "3.4.11", behavior: Object, event: null, ns: Object, time: Object…}
```
